src/session/content/message_row/sticker_picture.rs

```diff
@@ -7,7 +7,7 @@ mod imp {
     use once_cell::sync::Lazy;
     use std::cell::{Cell, RefCell};
 
-    const SIZE: i32 = 200;
+    const SIZE: i32 = 180;
```

Comment:
I'm gonna cherry-pick that change in a different branch to keep this clean from unrelated stuff.

Reply:
https://github.com/melix99/telegrand/pull/469.

Please rebase after it has been merged.

Reply:
Sure, can you make a quick review?

Reply:
doesn't have to be detailed